fortress/install_ubuntu.md

```diff
@@ -8,14 +8,14 @@ First install some necessary tools:
 
 ```bash
 sudo apt-get update
-sudo apt-get install lsb-release wget gnupg
+sudo apt-get install lsb-release gnupg
 ```
 
 Then install Ignition Fortress:
 
 
 ```bash
-sudo wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
+curl https://packages.osrfoundation.org/gazebo.gpg --output /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
```

Comment:
I think we need `sudo` here.

Reply:
good catch [90296fd](https://github.com/gazebosim/docs/pull/451/commits/90296fd6e99c581bc84c0895daa18ebe5e6e12b8)